test/overrides.test.ts

```diff
@@ -137,7 +137,7 @@ test('request overrides take precedence over adapter hardcoded ones', async (t)
     base: 'OVER1',
     quote: 'USD',
     overrides: {
-      test: {
+      test_adapter: {
```

Comment:
Can we also test uppercase override names (exact match the adapter name)?
```suggestion
      TEST_ADAPTER: {
```

Reply:
Added an extra test at the bottom.